ssh/tailssh/incubator.go

```diff
@@ -210,7 +274,7 @@ func startWithPTY(cmd *exec.Cmd, ptyReq ssh.Pty) (ptyFile *os.File, err error) {
 			tty.Close()
 		}
 	}()
-	ptyRawConn, err := ptyFile.SyscallConn()
+	ptyRawConn, err := tty.SyscallConn()
```

Comment:
whooops, I guess this mattered? did this fix something?

Reply:
nope :/